classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -120,16 +120,20 @@ function getDisplayStatus(&$navigationMenuItem) {
 				$navigationMenuItem->setIsDisplayed($context && ($context->getSetting('mailingAddress') || $context->getSetting('contactName')));
 				break;
 			case NMI_TYPE_USER_REGISTER:
+				$navigationMenuItem->setIsDisplayed(!$isUserLoggedIn && !$templateMgr->get_template_vars('disableUserReg'));
```

Comment:
Let's pull this from the context setting rather than relying on it being in the template manager: `$context->getSetting('disableUserReg')`

Reply:
Done